### PR TITLE
Juagargi/isd loops disallowed

### LINKED ISSUE #184: ISD loops
The code base disallows ISD loops by default now, but this behavior will change shortly.
Explicitly disallow them. Here is a hint:
- ISD loops filter: look at `go/beacon_srv/internal/beacon/policy.go`, `type Filter`, `AllowIsdLoop`.

### PR DESCRIPTION
**Pull request by juagargi**
Write configuration for nodes in infrastructure ASes to disallow ISD loops.
Closes #184

### CODE REVIEW

File: scionlab/scion_config.py [hunk 1]
```diff
@@ -123,6 +123,9 @@ def _gen_configs(self, cb):
 
         for service in self._control_services():
             if service.type == Service.CS:
+                if service.AS.is_infrastructure_AS():
```
Comment (FR4NK-W) [inline scionlab/scion_config.py]: >if service.AS.is_infrastructure_AS():

Is there a reason we are special casing this for infrastructure ASes?

If UserASes want a different configuration for some reason, they can still overwrite it, right?

File: scionlab/scion_config.py [hunk 2]
```diff
@@ -300,6 +307,13 @@ def build_cs_conf(self, service):
                 'resolution_fraction': 0.4,
             },
         })
+        if service.AS.is_infrastructure_AS():
+            conf['bs'].update({
```
Comment (FR4NK-W) [inline scionlab/scion_config.py]: Same here
>if service.AS.is_infrastructure_AS():

Would make the change a bit simpler

File: scionlab/scion_config.py (hunk 1 above)
Comment (juagargi) [reply scionlab/scion_config.py]: The disallow ISD loops inside the beaconing policy is necessary in the infrastructure nodes only. User ASes behave per default when beaconing, and they can of course modify their beaconing policy whenever they want. I don't have a strong opinion on limiting this policy to infrastructure ASes only, but I didn't see a reason to do have it also in user ASes.

Comment (FR4NK-W) [reply scionlab/scion_config.py]: >didn't see a reason to do have it also in user ASes

To not special case them (, and later having to revert it when introducing UserAS APs)

**Pull request merged**